docs/package_info/contributing.rst

```diff
@@ -6,7 +6,7 @@ an established consistent way of doing things. A consistent style increases
 readability, decreases bug-potential and makes it faster to understand how
 everything works together.
 
-PRAW follows :PEP:`8` and :PEP:`257`. You can use ``pre-push.py`` to test for
+PRAW follows :PEP:`8` and :PEP:`257`. You can use ``pre_push.py`` to test for
```

Comment:
Yes, there are a few other checks that it preforms. Maybe you could add "compliance with these PEP's and a few other checks" or something to that extent.